config/prod/prow/jobs/custom-jobs.yaml

```diff
@@ -291,44 +268,31 @@ periodics:
 
 postsubmits:
   knative/test-infra:
-  - name: post-knative-prow-config-updater
+  - name: post-knative-prow-cluster-config-updater
```

Comment:
Do we need 4 jobs? Can we just have one job updating all?

Reply:
I'd like to divide it at least to 2:
- prow cluster deployment (This will be deprecated pretty soon)
- prow build cluster deployment

Reply:
Actually I think you're right, combined to be a single job